app/controllers/api/v1/cities_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Api
+  module V1
+    class CitiesController < Api::V1::ApiController
+      def index
+        render json: response_data(country.cities), status: :ok
+      end
+
+      def create
+        new_city = country.cities.create!(cities_params)
+
+        render json: response_data(new_city), status: :created
+      end
+
+      def show
+        render json: response_data(city), status: :ok
+      end
+
+      def update
+        city.update!(cities_params)
+
+        render json: response_data(city), status: :ok
+      end
+
+      def destroy
+        city.destroy!
+
+        head :no_content
+      end
+
+      private
+
+      def cities_params
```

Comment:
```suggestion
      def city_params
```
Safe to ignore but I like it better 😆

Reply:
Updated